liip/imagine-bundle/1.7/etc/routing/imagine.yaml

```diff
@@ -0,0 +1,19 @@
+liip_imagine_filter: 
```

Comment:
That isn't something we can do until the next major release, as people rely on the current name for their own usage in pre-flex applications.